Libs/Core/ctkUtils.h

```diff
@@ -62,6 +62,17 @@ void CTK_CORE_EXPORT qListToSTLVector(const QStringList& list, std::vector<std::
 /// Convert a Vector of string to QStringList
 void CTK_CORE_EXPORT stlVectorToQList(const std::vector<std::string>& vector, QStringList& list);
 
+///
+/// \ingroup Core
+/// Convert a QStringList to a set
```

Comment:
Describe that this method is added so that the same code works for pre and post Qt-5.14.